config_types.cpp

```diff
@@ -263,3 +266,93 @@ const char* server_addr::get_last_error(void) const
 {
     return gai_strerror(m_last_error);
 }
+
+config_cpu_list::config_cpu_list(unsigned int cores)
+{
+    for (unsigned int i = 0; i < cores; ++i)
+        cpu_list.push_back(i);
+
+    next_cpu = cpu_list.begin();
+}
+
+config_cpu_list::config_cpu_list(std::string str)
```

Comment:
is this function deals with input like this - `0,5,8-11` , which is a valid input for taskset?

Reply:
changed the parsing function, this case is handled now